pkg/config/config.go

```diff
@@ -68,6 +68,9 @@ const (
 
 	terminationLogPathDefault = "/dev/termination-log"
 	terminationLogPathEnvVar  = "TERMINATION_LOG_PATH"
+
+	// environment variable for the Git rewrite setting
+	useGitRewriteRule = "ENABLE_GIT_REWRITE_RULE"
```

Comment:
Please document the new environment variable in https://github.com/shipwright-io/build/blob/main/docs/configuration.md. For consistency, I would also move `GIT_` to the beginning of the environment variable name.

Reply:
Done.